lib/ui/mockData/MockUserProfileViewModel.dart

```diff
@@ -68,27 +68,68 @@ class MockUserProfileListItemViewModel {
   }
 }
 
+class _LocalisedStrings {
+  static String switchLanguage() => Intl.message("Switch Language");
+
+  static String farmDetails() => Intl.message("Your Farm Details");
+
+  static String updatePin() => Intl.message("Update Pin");
+
+  static String createNewProfile() => Intl.message("Create New Profile");
+
+  static String inviteFriends() => Intl.message("Invite Friends");
+
+  static String privacyPolicy() => Intl.message("Privacy Policy");
+
+  static String termsOfUse() => Intl.message("Terms of Use");
+
+  static String deleteProfile() => Intl.message("Delete Profile");
+
+  static String dummyTitle1() =>
+      Intl.message("Lorem ipsum dolor sit amet, consectetur adipiscing elit.");
+
+  static String dummyTitle2() => Intl.message(
```

Comment:
lets not translate the mock. this will put a lot of unneeded string in the translation list.

Reply:
I will remove as well the `Intl.message` in methods `buildLager` and `build` in this class:
- `Intl.message(_mockActionTitleLarger[index])`
- `Intl.message( _mockActionTitle[index])`